Resources/Private/Language/locallang.xlf

```diff
@@ -115,17 +115,23 @@
 				<source>Create new Glossary translation</source>
 			</trans-unit>
 
+			<trans-unit id="glossary.sync.title">
+				<source>DeepL sync ready</source>
+			</trans-unit>
 			<trans-unit id="glossary.sync.message">
 				<source>Synchronization with DeepL is done. Your glossaries are ready to use.</source>
 			</trans-unit>
-			<trans-unit id="glossary.sync.title">
-				<source>DeepL Sync ready</source>
+			<trans-unit id="glossary.sync.title.invalid">
+				<source>DeepL glossary sync invalid</source>
 			</trans-unit>
-			<trans-unit id="glossary.not-sync.message">
-				<source>Glossary synchronization with DeepL is pending. Your glossaries are not operational.</source>
+			<trans-unit id="glossary.sync.message.invalid">
```

Comment:
trans-unit sections should contain id and res attributes. Let's take it for now, should be streamlined in a dedicated followup change.